app/models/miq_product_feature.rb

```diff
@@ -153,6 +153,58 @@ def self.seed_feature(hash, features)
     return feature, status
   end
 
+  def self.find_all_by_identifier(features)
+    where(:identifier => features)
+  end
+
+  def self.obj_features
+    @obj_cache ||= begin
```

Comment:
Could this cause test contamination?  If so, can you add it to EvmSpecHelper.clear_caches so it will be cleared on every test automatically?